drivers/sdcard/sdcard.py

```diff
@@ -44,28 +56,27 @@ def __init__(self, spi, cs):
 
     def init_card(self):
         # init CS pin
-        self.cs.high()
-        self.cs.init(self.cs.OUT_PP)
+        self.cs.init(self.cs.OUT, 1)
 
         # init SPI bus; use low data rate for initialisation
-        self.spi.init(self.spi.MASTER, baudrate=100000, phase=0, polarity=0)
+        self.spi.init(baudrate=100000, phase=0, polarity=0)
```

Comment:
I don't think thiss will work anymore on pyboard, since it requires the mode argument.  stmhal's SPI driver needs to be changed (in a backwards-incompatible way?) to support the new HW API.